connect/src/com/telenor/connect/WellKnownAPI.java

```diff
@@ -28,5 +33,44 @@ public String getIssuer() {
         public Set<String> getNetworkAuthenticationTargetIps() {
             return networkAuthenticationTargetIps;
         }
+
+        protected WellKnownConfig(Parcel in) {
+            issuer = in.readString();
+            networkAuthenticationTargetIps = new HashSet<>();
+            int count = in.readInt();
+            for (int i = 0; i < count; i++) {
+                networkAuthenticationTargetIps.add(in.readString());
+            }
+        }
+
+        @Override
+        public int describeContents() {
+            return 0;
+        }
+
+        @Override
+        public void writeToParcel(Parcel dest, int flags) {
+            dest.writeString(issuer);
+            if (networkAuthenticationTargetIps == null) {
```

Comment:
Done!

I mean, it's a bit tricky to make sure that the variable networkAuthenticationTargetIps is always non-null (I think null value will be injected if the SeralizedName is missing in JSON), but it's much easier to achieve the same with the getter (which I did :-)